fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -187,6 +184,14 @@ def self.run(options)
         end
       end
 
+      def self.puts_download_url(body)
+        version_url = body['public_url']
+        version_url = body['public_url'] + '/app_versions/' + body['config_url'].split('/').last if body['config_url']
```

Comment:
These lines has more potential nil errors
As this method only prints convenient information, perhaps result parsing problems could be captured and printed instead of raising fatal errors?